DoSomething/gladiator
# Adding/updating migrations for the database structure.

Fixes #2

#### What's this PR do?
This PR updates/add migrations to setup the database structure that we discussed as a team.

#### Where should the reviewer start?
Check out the migrations in the **/database** directory.

#### How should this be manually tested?
Pull down the branch and run `php artisan migrate` and make sure you don't get any errors! If you already had the old migrations setup, make sure to `php artisan migrate:reset` or clear out your database before pulling down the branch!

#### What are the relevant tickets?
#2 

#### Questions:
Are there any additional fields that should be treated as `->index()` for lookups, etc? (specifically looking at you @angaither :wink: )

Also, apart from the `northstar_id` in the `users` table, should we also have one for `phoenix_id`?

---
@angaither @sbsmith86 

cc: @DFurnes @deadlybutter
